<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>537</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>QtHW</string>
  </property>
  <widget class="QLabel" name="csv_prompt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose CSV file:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run_command">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze Loaded CSV</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="request_csv">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Frequency Domain CSV</string>
   </property>
  </widget>
  <widget class="QToolButton" name="select_file">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmation">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No CSV Loaded</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_data">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_rate">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desired Number of DSP Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sr_label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Rate:</string>
   </property>
  </widget>
  <widget class="QWidget" name="select_done" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: yellow</string>
   </property>
  </widget>
  <widget class="QLabel" name="field_warning">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>869</width>
     <height>78</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Digital Signal Analyser&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="settings_view">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="record_audio_request">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Your Own Audio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWidget.qrc"/>
 </resources>
 <connections/>
</ui>
